src/test/java/uk/gov/hmcts/cmc/claimstore/models/serialization/CountyCourtJudgmentSerializationTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.cmc.claimstore.models.serialization;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.InvalidTypeIdException;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.config.JacksonConfiguration;
+import uk.gov.hmcts.cmc.claimstore.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleCountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleRepaymentPlan;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleTheirDetails;
+import uk.gov.hmcts.cmc.claimstore.utils.ResourceReader;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CountyCourtJudgmentSerializationTest {
+
+    private static final ObjectMapper mapper = new JacksonConfiguration().objectMapper();
+
+    @Test
+    public void shouldConvertCCJJsonToJava() throws IOException {
+
+        //given
+        TheirDetails defendant = SampleTheirDetails.builder().withRepresentative(null).individualDetails();
```

Comment:
does anyone else not like the pattern of optional fields automatically being populated by the builder?

Reply:
Well, I wouldn't mind if this is irrelevant (do not affect anything). In this case I'd change it.

Updated

Reply:
to be clear I meant the `. withRepresentative(null)` bit

Reply:
Updated here:
```
CompanyDetails defendant = SampleTheirDetails.builder().companyDetails();
```

and in other tests. representative is null by default now.